packages/codegen/src/openapi.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 Cognite AS
+import { OpenAPIV3 } from 'openapi-types';
+
+export type OpenApiDocument = OpenAPIV3.Document;
+export type OpenApiSchema = OpenAPIV3.SchemaObject;
+export type OpenApiPathItem = OpenAPIV3.PathItemObject;
+export type OpenApiOperationObject = OpenAPIV3.OperationObject;
+export type OpenApiPathsObject = OpenAPIV3.PathsObject;
+
+export interface FilteredPathsObject {
+  [pattern: string]: OpenApiPathItem;
+}
+
+export function isReferenceObject(
+  item: any
+): item is OpenAPIV3.ReferenceObject {
+  return '$ref' in item;
+}
+
+export function filterPaths(paths: OpenApiPathsObject): FilteredPathsObject {
+  return Object.keys(paths)
+    .filter((k) => typeof paths[k] !== 'undefined')
+    .reduce((acc, key) => {
+      return Object.assign(acc, { [key]: paths[key] });
+    }, {} as FilteredPathsObject);
+}
+
+export function operationsInPaths(
+  paths: OpenApiPathsObject
+): OpenApiOperationObject[] {
+  const filtered = filterPaths(paths);
+  return Object.values(filtered)
+    .map(operationsInPath)
+    .reduce((acc, v) => acc.concat(v), []);
+}
```

Comment:
These seem unused?

Reply:
See codegen.ts line 88

Reply:
Still unused. Are you mixin this with the next method `operationsInPath` ?